ocaml/xapi/import_raw_vdi.ml

```diff
@@ -71,17 +71,26 @@ let localhost_handler rpc session_id vdi_opt (req: Request.t) (s: Unix.file_desc
                  error "Not enough info supplied to import";
                  raise (HandleError(Api_errors.Server_error(Api_errors.internal_error, ["Not enough info supplied to import"]),Http.http_400_badrequest ~version:"1.0" ()))
              in
-             let prezeroed = not (Sm_fs_ops.must_write_zeroes_into_new_vdi ~__context vdi) in
-             Sm_fs_ops.with_block_attached_device __context rpc session_id vdi `RW
-               (fun path ->
-                  let headers = Http.http_200_ok ~keep_alive:false () @
-                                [ Http.Hdr.task_id ^ ":" ^ (Ref.string_of task_id);
-                                  content_type ] in
-                  Http_svr.headers s headers;
-                  if chunked
-                  then Vhd_tool_wrapper.receive (Vhd_tool_wrapper.update_task_progress __context) "raw" "chunked" s None path "" prezeroed
-                  else Vhd_tool_wrapper.receive (Vhd_tool_wrapper.update_task_progress __context) (Importexport.Format.to_string format) "none" s req.Request.content_length path "" prezeroed
-               );
+             let headers = Http.http_200_ok ~keep_alive:false () @
+                           [ Http.Hdr.task_id ^ ":" ^ (Ref.string_of task_id);
+                             content_type ] in
+             Http_svr.headers s headers;
```

Comment:
There is a small change in behaviour here: we'll send these headers, even if we fail to attach the VDI afterwards.
But this is probably not a big deal, because we can also fail during import.

Reply:
I think this comment was meant for the previous patch hunk. You could make a `send_headers` function and call it from both `copy` where it used to be called previously, and in the `Tar` branch as well.

Reply:
My question is, is that really necessary, or is this slight behaviour change okay? (As we can fail later during import anyway) - I don't want to complicate the code if it's not necessary.